contrib/intel/jenkins/build.py

```diff
@@ -28,6 +28,8 @@ def build_libfabric(libfab_install_path, mode, cluster=None, ucx=None):
         prov_list = common.daos_prov_list
     elif (cluster == 'gpu'):
         prov_list = common.gpu_prov_list
+    elif (ucx):
+        prov_list = common.ucx_prov_list
```

Comment:
what is the reson to have this? are other providers going to be added tot this list? as per the design of the prov_list is meant to hold multiple providers applicable for an application/middleware. it doesn't seem like that in ucx case.

Reply:
Agreed, I reworked it to use the default prov list and when it iterates through it it only adds ucx if the ucx flag is set.